styleguide/source/_patterns/05-pages/topic.twig

```diff
@@ -1,18 +1,34 @@
 {% extends '@templates/three-column.twig' %}
 
+{% block header %}
+  {% include '@organisms/ribbon.twig' %}
+  {% include '@organisms/wayfinder.twig' %}
+  <div class="container">
+    {% include '@atoms/page-title.twig' %}
+  </div>
```

Comment:
Please add a new region to the template of `contentTop` that is full-width of the layout and put the page title there. This should match the work that @chelsiejohnston is doing on the Two Column Layout.

Reply:
I have created a new ticket - [EWL-4537](https://issues.ama-assn.org/browse/EWL-4537) to capture this change.